README.md

```diff
@@ -104,10 +104,75 @@ To get specific instructions for your preferred shell run:
 mongocli completion <bash|zsh|fish|powershell> --help
 ```
 
+## AtlasCLI (Pre-Release)
+![GO tests](https://github.com/mongodb/mongocli/workflows/GO%20tests/badge.svg)
+![golangci-lint](https://github.com/mongodb/mongocli/workflows/golangci-lint/badge.svg)
+
+The MongoDB AtlasCLI is a modern command line interface that enables you to manage MongoDB Atlas from the terminal.
+
+![atlascli-atlas-quickstart](https://user-images.githubusercontent.com/5663078/156184669-57c8ddce-6f0a-4e84-9311-2d996cb27942.gif)
+
+## Installing
+
+AtlasCLI is currently in the pre-release phase, so it should not be used in production environment. 
+
+### Pre-built Binaries
+
+Download the appropriate version for your platform from [AtlasCLI releases](https://github.com/mongodb/mongocli/releases).
+Once downloaded, the binary can be run from anywhere.
```

Comment:
After you download the library, you can run it from anywhere and don't need to install it into a global location.